velox/common/base/Status.h

```diff
@@ -537,6 +605,41 @@ inline Status genericToStatus(Status&& st) {
   return std::move(st);
 }
 
+struct CompileTimeEmptyString {
```

Comment:
Is this a copy-paste from somewhere else? Would it be possible to refactor to avoid duplicating code?